### ISSUE TITLE
Crypter cuts strings

### ISSUE THREAD

**Issue by MaximBalaganskiy** [FastReports/FastReport#344]
Test case
```csharp
var connection = "Server=azuresqlserver.database.windows.net;Database=azuredatabase;Authentication=Active Directory MSI;User Id=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx;"
if(connection != Crypter.DecryptString(Crypter.EncryptString(connection))) {
  throw new Exception();
};
```

Cost me two days to figure out why previewing reports does not work - caching serializes and deserializes connection strings and encryption ruins them.

**Comment by Detrav** [comment]
I've tested the current version on .net 5, this code does not throw an error.

**Comment by MaximBalaganskiy** [comment]
try .net6

**Comment by Detrav** [comment]
Tried .net 6, got the error.
![image](https://user-images.githubusercontent.com/5013291/141464500-185385fb-5951-4731-a265-5d37bd407cfc.png)

**Comment by fediachov** [comment]
https://github.com/dotnet/runtime/issues/61398

**Comment by Detrav** [comment]
It looks like the problem is that the encrypted stream is not fully read, we will fix it in FR crypter in the next update.

**Comment by fediachov** [comment]
Hi,

The bug has been fixed in a build [2021.4.15 ](https://www.nuget.org/packages/FastReport.OpenSource/)
Thanks for bug report!
